### PR TITLE
[#666] Configure surefire plugin for plain Maven

### PR DESCRIPTION
**Pull request by kthoms**
The surefire plugin needs additional dependencies to execute JUnit 5
tests.

Signed-off-by: Karsten Thoms <karsten.thoms@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/TestProjectDescriptor.xtend [hunk 1]
```diff
@@ -190,6 +190,26 @@ abstract class TestProjectDescriptor extends ProjectDescriptor {
...
+								<!-- required to execute JUnit 5 tests -->
+								<plugin>
+									<groupId>org.apache.maven.plugins</groupId>
+									<artifactId>maven-surefire-plugin</artifactId>
+									<version>2.21.0</version>
+									<dependencies>
+										<dependency>
+											<groupId>org.junit.platform</groupId>
+											<artifactId>junit-platform-surefire-provider</artifactId>
+											<version>1.2.0</version>
```
Comment (cdietrich) [inline org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/TestProjectDescriptor.xtend]: does this match 5.1.0

Comment (kthoms) [reply org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/TestProjectDescriptor.xtend]: It works with 5.1.0

Comment (cdietrich) [reply org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/TestProjectDescriptor.xtend]: ok

**Comment by kthoms** [comment]
```
[INFO] --- maven-surefire-plugin:2.21.0:test (default-test) @ org.xtext.example.mydsl3.tests ---
[INFO]
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.xtext.example.mydsl3.tests.MyDslParsingTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.224 s - in org.xtext.example.mydsl3.tests.MyDslParsingTest
[INFO]
[INFO] Results:
[INFO]
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0
```

**Pull request merged**